src/checkValidTargetFilters.ts

```diff
@@ -38,13 +38,13 @@ export async function checkValidTargetFilters(options: Options): Promise<void> {
       return reject(new FirebaseError("Cannot specify both --only and --except"));
     }
     const nonFilteredTypes = VALID_DEPLOY_TARGETS.filter(
-      (t) => !["hosting", "functions"].includes(t)
+      (t) => !["hosting", "functions", "firestore"].includes(t)
```

Comment:
Hi @bkendall, just want to chime in. Does it also support storage? 

Our pipeline also deploy storage rules with --only

```
firebase deploy --only storage:rules 
```